firecrest/AsyncClient.py

```diff
@@ -185,7 +194,9 @@ def __init__(
         #: client and the rate will be controlled by the request rate of the
         #: `tasks` microservice.
         self.polling_sleep_times: list = 250 * [0]
-        self._api_version: Version = parse("1.13.1")
+        #: Disable all logging from the client.
+        self.disable_client_logging: bool = False
+        self._api_version: Version = parse("1.15.0")
```

Comment:
I also updated the default api version since this is the production version on CSCS.